src/js/messaging/containers/Compose.jsx

```diff
@@ -163,6 +166,7 @@ class Compose extends React.Component {
 
 const mapStateToProps = (state) => {
   return {
+    persistFolder: state.folders.data.currentItem.persistFolder,
```

Comment:
It was. In department-of-veterans-affairs/messaging-fe#63, it's supposed to go return back to the folder the veteran clicked "Compose" from, but I want to push back on that a bit. That seems like a weird flow for the user.